cli/smartcontract/testdata/update/updated.go

```diff
@@ -0,0 +1,91 @@
+package deploy
+
+import (
+	"github.com/nspcc-dev/neo-go/cli/smartcontract/testdata/deploy/sub"
+	"github.com/nspcc-dev/neo-go/pkg/interop"
+	"github.com/nspcc-dev/neo-go/pkg/interop/contract"
+	"github.com/nspcc-dev/neo-go/pkg/interop/iterator"
+	"github.com/nspcc-dev/neo-go/pkg/interop/runtime"
+	"github.com/nspcc-dev/neo-go/pkg/interop/storage"
+)
+
+var key = "key"
+
+const mgmtKey = "mgmt"
+
+func _deploy(data any, isUpdate bool) {
+	var value string
+
+	ctx := storage.GetContext()
+	if isUpdate {
+		value = "on update"
+	} else {
+		value = "on create"
+		sh := runtime.GetCallingScriptHash()
+		storage.Put(ctx, mgmtKey, sh)
+
+		if data != nil {
+			arr := data.([]any)
+			for i := 0; i < len(arr)-1; i += 2 {
+				storage.Put(ctx, arr[i], arr[i+1])
+			}
+		}
+	}
+
+	storage.Put(ctx, key, value)
+}
+
+// Fail just fails.
+func Fail() {
+	panic("as expected")
+}
```

Comment:
Useless method. Reduce this contract to the minimum required functionality that is necessary for update test only.